var/spack/repos/builtin/packages/py-pbr/package.py

```diff
@@ -29,8 +29,10 @@ class PyPbr(PythonPackage):
     """PBR is a library that injects some useful and sensible default
        behaviors into your setuptools run."""
     homepage = "https://pypi.python.org/pypi/pbr"
-    url      = "https://pypi.python.org/packages/source/p/pbr/pbr-1.8.1.tar.gz"
+    url      = "https://pypi.io/packages/source/p/pbr/pbr-1.10.0.tar.gz"
 
     version('1.8.1', 'c8f9285e1a4ca6f9654c529b158baa3a')
+    version('1.10.0', '8e4968c587268f030e38329feb9c8f17')
```

Comment:
Can you add newer versions to the top? It helps to keep things in descending order.